src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/OneVersusAllTrainer.cs

```diff
@@ -227,24 +222,25 @@ public override MulticlassPredictionTransformer<OneVersusAllModelParameters> Fit
                         var transformer = TrainOne(ch, Trainer, td, i);
                         featureColumn = transformer.FeatureColumnName;
                     }
+                    predictors[i] = TrainOne(ch, Trainer, td, i).Model as T;
```

Comment:
```suggestion
                    predictors[i] = (T)TrainOne(ch, Trainer, td, i).Model;
```

Reply:
This also happens on line 152, I will fix it in both locations.